README.md

```diff
@@ -16,5 +16,13 @@ gcloud compute instances create $USER-cos-gpu-test \
     --accelerator=type=nvidia-tesla-k80 \
     --boot-disk-size=25GB \
     --maintenance-policy=TERMINATE \
-    --metadata-from-file "user-data=install-test-gpu.cfg"
+    --metadata-from-file "cos-gpu-installer-env=/tmp/gpu-installer-env,user-data=install-test-gpu.cfg,run-installer-script=scripts/run_installer.sh,run-cuda-test-script=scripts/run_cuda_test.sh"
```

Comment:
It should be `scripts/` instead of `tmp/` for `gpu-installer-env`.

Reply:
Good catch! Thanks!